platform_plugin_saleor/views.py

```diff
@@ -0,0 +1,28 @@
+"""Generic views for the platform plugin saleor."""
+
+from os.path import dirname, realpath
+from subprocess import CalledProcessError, check_output
+
+from django.http import JsonResponse
+
+import platform_plugin_saleor
```

Comment:
```suggestion
from platform_plugin_saleor import __version__ as plaform_plugin_version
```
What do you think of this line, to import only the version and not all init module?